src/main/java/com/ncc/aif/ValidateAIF.java

```diff
@@ -287,6 +303,18 @@ private static boolean processArgs(String[] args, Set<ArgumentFlags> flags, Set<
             logger.error("Please specify either file(s) or a directory of files to validate.");
             return false;
         }
+        if (abortStr != null) {
```

Comment:
Why not add this code above (line 258)? Readability?

Reply:
Yeah, basically.  I try not to have too much code in each case of a `switch`.  So I'm just collecting whatever additional info I need immediately (such as parameters after certain options like the `[num]` after `--abort` and the files/directories after `-f` and `-d`, fail on some quick/obvious errors, and do the rest after the `switch` is over.